libs/blocks/global-navigation/dark-nav.css

```diff
@@ -0,0 +1,492 @@
+  --feds-background-footer--dark: #1B1B1B;
+  --feds-borderColor-featuredProducts--dark: #999;
+}
+
+.feds-cta--primary {
+  background-color: #066ce7;
+  border-color: #066ce7;
+  color: rgb(255, 255, 255);
+}
+
+.darknav .feds-navLink:hover,
+.darknav .feds-navLink:focus {
+  color: var(--feds-color-link--hover--dark);
+}
+
+.darknav .feds-navLink--hoverCaret:hover,
+.darknav .feds-navLink--hoverCaret:focus {
+  background-color: #151515;
+  color: var(--feds-color-mobile-headline--dark);
+}
+
+.darknav .feds-navLink {
+  color: var(--feds-color-mobile-link--dark);
+}
+
+.darknav .feds-navLink--hoverCaret:after,
+.darknav  .feds-navLink[class *= '-gradient'] .feds-navLink-title:after {
+  border-color: var(--feds-color-link--dark);
+}
+
+.darknav .feds-navLink-description {
+  color: var(--feds-color-navLink-description--dark);
+}
+
+.darknav .feds-navLink:hover .feds-navLink-description,
+.darknav .feds-navLink:focus .feds-navLink-description {
+  color: var(--feds-color-navLink-description--dark);
+}
+
+.darknav .feds-navLink--blue {
+  color: var(--feds-color-blue-link--dark);
+}
+
+.darknav .feds-navLink:focus-visible,
+.darknav .feds-search-trigger:focus-visible,
+.darknav .feds-profile-button:focus-visible,
+.darknav .feds-logo:focus-visible,
+.darknav .feds-search-input:focus-visible {
+  outline-style: solid;
+}
+
+/* Desktop styles */
+@media (min-width: 900px) {
+  .darknav .feds-navLink--hoverCaret:hover,
+  .darknav .feds-navLink--hoverCaret:focus {
+    background-color: #111;
+    color: var(--feds-color-link--dark);
+  }
+
+  .darknav .feds-navLink {
+    color: var(--feds-color-link--dark);
+  }
+
+  .darknav .feds-navLink[class *= '-gradient']:hover .feds-navLink-title:after,
+  .darknav .feds-navLink[class *= '-gradient']:focus .feds-navLink-title:after {
+    border-color: var(--feds-color-link--hover--dark);
+  }
+
+  .darknav .feds-navItem--active > .feds-navLink:only-child:hover,
+  .darknav .feds-navItem--active > .feds-navLink:only-child:focus {
+    color: var(--feds-color-link--dark);
+  }
+  .darknav .feds-navItem--active > .feds-navLink:before {
+    border-bottom: 2px solid #DBDBDB;
+  }
+}
+
+/* global-navigation.css */
+header.global-navigation .darknav {
+  background-color: var(--feds-background-mobile-nav--dark);
+}
+
+.darknav .feds-topnav-wrapper {
+  background-color: var(--feds-background-mobile-nav--dark);
+}
+
+.darknav .feds-nav-wrapper {
+  border-top: 1px solid #323232;
+  background-color: var(--feds-background-mobile-nav--dark);
+}
+
+.feds-toggle {
+  color: #fff;
+}
+
+.darknav .feds-cta--secondary {
+  background-color: #111;
+  border-color: #393939;
+  color: var(--feds-color-link--dark);
+}
+
+.darknav .feds-cta--secondary:hover,
+.darknav .feds-cta--secondary:focus {
+  border-color: #444;
+  background-color: #2C2C2C;
+  color: var(--feds-color-link--dark);
+}
+
+.darknav .feds-breadcrumbs a {
+  color: #B0B0B0;
+}
+
+.darknav .feds-breadcrumbs [aria-current] {
+  color: #DBDBDB;
```

Comment:
There seem to be plenty of colors repeated throughout the file that already have variables associated with them. For example here `#DBDBDB` is `--feds-color-link--dark`/`--feds-color-headline--dark`. You may or may not want to replace these repetitions with variables depending on whether changing one color should change the other instances of the same color.